vignettes/colors.Rmd

```diff
@@ -148,8 +148,15 @@ nms <- dplyr::filter(get_cmapplot_global("palettes"),  type == "sequential")$nam
 purrr::walk(nms, viz_gradient)
 ```
 
+If you use a sequential palette for a discrete scale, the package will automatically choose colors from across the selected gradient and interpolate additional ones if needed. For example, the `blues` palette is shown below, interpolating nine colors from the initial palette list of seven.
+
+```{r show_sequential_discrete, echo = FALSE, fig.asp = 300/670}
+nms <- fetch_pal("blues")
+viz_palette("blues",num = 9)
```

Comment:
@dlcomeaux I'm OK with this, I think. I appreciate the value of this clarification, but I don't *love* that it's in the section that is intended as a menu of options as opposed to a description of how something works. What would you think about moving this up to the `### Discrete scales` section up top, and combining the `viz_palette` call that demonstrates the interpolation with an actual example of a ggplot with a discrete scale of a continuous palette? Something like [Timi's here](https://github.com/CMAP-REPOS/cmapplot/issues/119#issue-820350735). 

Also, does the `nms` construction here have a purpose?

Reply:
Oops, the nms was a holdover. That should be cut.